src/System.Management.Automation/engine/runtime/Operations/MiscOps.cs

```diff
@@ -568,14 +564,22 @@ private static string GetParameterText(string parameterName)
                 var sb = ScriptBlock.Create(updatedScriptblock.ToString());
                 var commandInfo = new CmdletInfo("Start-Job", typeof(StartJobCommand));
                 commandProcessor = context.CommandDiscovery.LookupCommandProcessor(commandInfo, CommandOrigin.Internal, false, context.EngineSessionState);
-                var parameter = CommandParameterInternal.CreateParameterWithArgument(
+                var workingDirectoryParameter = CommandParameterInternal.CreateParameterWithArgument(
+                    parameterAst: pipelineAst,
```

Comment:
perhaps make all the arguments passed in named parameters then (i.e `argumentName: argumentValue`)